packages/devextreme/scss/widgets/fluent/form/_sizes.scss

```diff
@@ -9,24 +9,30 @@ $fluent-form-group-caption-font-size: null !default;
 $fluent-form-group-content-padding-bottom: null !default;
 $fluent-form-default-top-padding: null !default;
 
-$fluent-form-label-font-size: 12px !default;
-$fluent-form-label-icon-size: 14px !default;
+$fluent-form-label-font-size: null !default;
+$fluent-form-label-icon-size: null !default;
 
 
 @if $size == "default" {
   $fluent-form-group-caption-bottom-margin: 6px !default;
-  $fluent-form-group-caption-bottom-padding: 19px !default;
-  $fluent-form-top-label-padding: 16px 0 3px !default;
+  $fluent-form-group-caption-bottom-padding: 20px !default;
+  $fluent-form-top-label-padding: 0 !default;
   $fluent-form-group-caption-font-size: 20px !default;
   $fluent-form-group-content-padding-bottom: 20px !default;
-  $fluent-form-default-top-padding: 26px !default;
+  $fluent-form-default-top-padding: 24px !default;
+
+  $fluent-form-label-font-size: 14px !default;
+  $fluent-form-label-icon-size: 14px !default;
 }
 
 @else if $size == "compact" {
   $fluent-form-group-caption-bottom-margin: 4px !default;
   $fluent-form-group-caption-bottom-padding: 10px !default;
```

Comment:
I suggest that paddings should be 8px here. Check please other values of variables for compact and default sizes